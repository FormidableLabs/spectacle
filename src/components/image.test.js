import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import Image from './image';

describe('<Image />', () => {
  test('should render correctly.', () => {
    const context = { styles: { components: { image: {} } } };
    const wrapper = mount(
      <Image
        src="foo.png"
        display="inline-block"
        width={2560}
        height={1440}
      />, { context });
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});
