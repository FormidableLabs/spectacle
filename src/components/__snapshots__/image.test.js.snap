// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> should render correctly. 1`] = `
<Image
  display="inline-block"
  height={1440}
  src="foo.png"
  width={2560}
>
  <img
    data-radium={true}
    src="foo.png"
    style={
      Object {
        "display": "inline-block",
        "height": 1440,
        "width": 2560,
      }
    }
  />
</Image>
`;
